src/main/java/org/ozwillo/dcexporter/service/DcModelMappingService.java

```diff
@@ -93,4 +93,13 @@ public DcModelMapping getById(String id) {
             return new AuditLogWapper(dcModelMapping, auditLog, datasetUrl, resourceUrl);
         }).collect(Collectors.toList());
     }
+
+    public Either<String, DcModelMapping> deleteById(String id) {
+        DcModelMapping dcModelMapping = dcModelMappingRepository.findById(id);
+        if ( dcModelMapping == null) return Either.left("Cette synchronisation de ressource n'existe pas");
+        else if (dcModelMappingRepository.deleteById(id) > 0) {
+            return Either.right(dcModelMapping);
+        }
+        else return Either.left("Aucune synchronisation de ressource supprimée");
```

Comment:
In this case, the resource is found, but for an unknown reason it can't be deleted.
Seems likely to be a server error (5XX)